docs/source/deep_dives/checkpointer.rst

```diff
@@ -240,27 +238,21 @@ The following snippet explains how the MetaCheckpointer is setup in torchtune co
         _component_: torchtune.training.FullModelMetaCheckpointer
 
         # directory with the checkpoint files
-        # this should match the output_dir above
+        # this should match the folder you used when downloading the model
         checkpoint_dir: <checkpoint_dir>
```

Comment:
Should we just explicitly change this to `/tmp/Llama-3.2-3B-Instruct` to match what's in the tune download command?